<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsComponent</class>
 <widget class="QTabWidget" name="CSettingsComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tb_SettingsOverview">
   <attribute name="title">
    <string>Overview</string>
   </attribute>
   <layout class="QGridLayout" name="gl_Overview">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="4" column="0">
     <widget class="QPushButton" name="pb_Network">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Network</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <spacer name="vs_OverviewBottom">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pb_Servers">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Servers</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pb_Audio">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Audio</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pb_Gui">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>GUI</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pb_Hotkeys">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Hotkeys</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pb_DataLoadAndCaches">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Data load and caches</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <spacer name="vs_OverviewTop">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="lbl_Hint">
      <property name="text">
       <string>Hint: see tooltips for shortcuts.</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="pb_Advanced">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Advanced</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="pb_Matching">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Matching</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="pb_SimulatorMessages">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Simulator (messages)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="pb_SimulatorBasics">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Simulator (basics)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="pb_Simulator">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Simulator</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsTrafficNetworkServers">
   <attribute name="title">
    <string>Servers</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsTrafficNetworkServers">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsNetworkServersComponent" name="comp_SettingsServersComponent">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="vs_SettingsTrafficNetworkServers">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsGui">
   <attribute name="title">
    <string>GUI</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsGui">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsGuiComponent" name="comp_SettingsGuiGeneral"/>
    </item>
    <item>
     <widget class="BlackGui::Components::CSettingsViewUpdateTimes" name="comp_SettingsGuiRefreshTimes"/>
    </item>
    <item>
     <widget class="QWidget" name="qw_SettingsGuiFont" native="true">
      <layout class="QHBoxLayout" name="hl_SettingsGuiFontColor">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsTrafficNetwork">
   <attribute name="title">
    <string>Network</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsTrafficNetwork">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsNetworkComponent" name="comp_SettingsNetwork">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsHotkeys">
   <attribute name="title">
    <string>Hotkeys</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsHotkeys">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item alignment="Qt::AlignTop">
     <widget class="BlackGui::Components::CSettingsHotkeyComponent" name="comp_SettingsHotkeys">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsAudio">
   <attribute name="title">
    <string>Audio</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsAudio">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item alignment="Qt::AlignTop">
     <widget class="BlackGui::Components::CAudioNotificationComponent" name="comp_AudioNotification">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_DataLoad">
   <attribute name="title">
    <string>Data and caches</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_DataLoad">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QScrollArea" name="sa_DataLoad">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="wi_DataLoadScrollable">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>203</width>
         <height>336</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vl_DataLoadOverview">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item alignment="Qt::AlignTop">
         <widget class="BlackGui::Components::CDbLoadOverviewComponent" name="comp_DataLoadOverview">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>250</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsSimulator">
   <attribute name="title">
    <string>Simulator</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsSimulator">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsSimulatorComponent" name="comp_SettingsSimulator">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsSimulatorBasics">
   <attribute name="title">
    <string>Simulator (basics)</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsSimulatorBascis">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsSimulatorBasicsComponent" name="comp_SettingsSimulatorBasics">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="vs_SettingsSimulatorBasics">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsSimulatorMessages">
   <attribute name="title">
    <string>Simulator (messages)</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsSimulatorMessages">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsSimulatorMessagesComponent" name="comp_SettingsSimulatorMessages"/>
    </item>
    <item>
     <spacer name="vs_SettingsSimulatorMessages">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>35</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_Matching">
   <attribute name="title">
    <string>Matching</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_Matching">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsMatchingComponent" name="comp_SettingsMatching">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_SettingsAdvanced">
   <attribute name="title">
    <string>Advanced</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_SettingsAdvanced">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="BlackGui::Components::CSettingsAdvancedComponent" name="comp_SettingsAdvanced">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CDbLoadOverviewComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbloadoverviewcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsSimulatorBasicsComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingssimulatorbasicscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsGuiComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingsguicomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsAdvancedComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingsadvancedcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsSimulatorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingssimulatorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsNetworkServersComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingsnetworkserverscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsNetworkComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingsnetworkcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsHotkeyComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingshotkeycomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsSimulatorMessagesComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingssimulatormessagescomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsViewUpdateTimes</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingsviewupdatetimes.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSettingsMatchingComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingsmatchingcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CAudioNotificationComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/audionotificationcomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pb_Servers</tabstop>
  <tabstop>pb_Gui</tabstop>
  <tabstop>pb_Network</tabstop>
  <tabstop>pb_Hotkeys</tabstop>
  <tabstop>pb_Audio</tabstop>
  <tabstop>pb_DataLoadAndCaches</tabstop>
  <tabstop>pb_Simulator</tabstop>
  <tabstop>pb_SimulatorBasics</tabstop>
  <tabstop>pb_SimulatorMessages</tabstop>
  <tabstop>pb_Matching</tabstop>
  <tabstop>pb_Advanced</tabstop>
  <tabstop>sa_DataLoad</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
